OCS/deploy-ocs.sh

```diff
@@ -4,15 +4,15 @@ LOCALNAMESPACE="local-storage"
 
 # export is required for envsubst
 export rook_override="${rook_override:-false}"
-export public_network="${public_network:-1.1.1.1/24}"
-export cluster_network="${cluster_network:-1.1.1.1/24}"
+export public_network="$(ip r | awk '/dev\ baremetal\ proto\ kernel\ scope\ link\ src/ {print $1}')"
```

Comment:
those variables probably only make sense when one has several nics, so those wouldnt be the default bridges

Reply:
Taking the current state, on the master nodes there are 2 interfaces:

eno1: same subnet as the provisioning bridge on the deploy host
brext(including eno2 nic): same subnet as the baremetal bridge on the deploy host

What should we set the public_network and cluster_network vars to in this case?

Reply:
those variables only make sense when one wants to segregate storage traffic, which requires a more advanced network configuration that what we currently have in place for htb.
even if we had it, i dont know if we would grab them from the current network config of the helper node (for htb, surely not, maybe after)

Reply:
But a segregated network is what was planned for HTB